Sample.Api/Tests/AirportServiceTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using Sample.Api.App;
+using Sample.Api.Model;
+
+namespace Sample.Api.Tests {
+	[TestFixture]
+	public class AirportServiceTests {
+		private AirportService _service;
+

```

Comment:
A good Service test should use Mock IHttpClientFactory object:
```suggestion
private IHttpClientFactory GetMockFactory(params string[] cities)
{
	var mockFactory = new Mock<IHttpClientFactory>();
	var mockHttpMessageHandler = new Mock<HttpMessageHandler>();
	mockHttpMessageHandler.Protected()
	    .Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.IsAny<HttpRequestMessage>(), ItExpr.IsAny<CancellationToken>())
	    .ReturnsAsync(new HttpResponseMessage
	    {
	        StatusCode = HttpStatusCode.OK,
	        Content = new StringContent("... build a valid JSON response object from REST service based on incomming parameters ..."),
	    });
 
	var client = new HttpClient(mockHttpMessageHandler.Object);
	mockFactory.Setup(f => f.CreateClient(It.IsAny<string>())).Returns(client);
	return mockFactory;
}

```